lib/versioning/semver-ruby/index.js

```diff
@@ -0,0 +1,74 @@
+const rubySemver = require('@snyk/ruby-semver');
+const { parse: parseVersion } = require('./version');
+const { ltr, parse: parseRange } = require('./range');
+const { isSingleOperator, isValidOperator } = require('./operator');
+const { pin, bump, replace } = require('./strategies');
+
+const getMajor = version => parseVersion(version).major;
+const getMinor = version => parseVersion(version).minor;
+const getPatch = version => parseVersion(version).patch;
+
+const isVersion = version => !!rubySemver.valid(version);
+const isEqual = (left, right) => rubySemver.eq(left, right);
+const isGreaterThan = (left, right) => rubySemver.gt(left, right);
+const isStable = version =>
+  parseVersion(version).prerelease ? false : isVersion(version);
```

Comment:
Nope, it will return false. We [have test](https://github.com/renovatebot/renovate/blob/fa275a005f84239d57cbe006e2779dbb9263a3ed/test/versioning/semver-ruby.spec.js#L145-L150) for such case.